lib/virt_feature_test_base.pm

```diff
@@ -47,13 +47,70 @@ sub run_test {
     die('Please override this subroutine in children modules to run desired tests.');
 }
 
+sub prepare_run_test {
+    my $self = shift;
+
+    my $wait_script              = "600";
+    my $logs_cleanup_script_name = "clean_up_virt_logs.sh";
+    my $logs_cleanup_script_url  = data_url("virt_autotest/$logs_cleanup_script_name");
+
+    script_run("source /usr/share/qa/qa_test_virtualization/shared/standalone");
+    my $download_bash_script = "curl -s -o ~/$logs_cleanup_script_name $logs_cleanup_script_url";
```

Comment:
Can all downloaded scripts from openqa data directory be put into a more specific directory like ~/downloaded_openqa_data_scripts?

Reply:
What does this serve ? Just try to unify something ? It is actually even not unification, because it just add another folder layer. It makes you feel good, but nothing better or changes. You should know all script employed by openqa api script_output resides in /tmp folder

Reply:
A code proposer can not ask all users be so familiar with all design details. Leaving all users including yourself, an easier interface/unified directory can always make debug easier.

Reply:
1. I do not think it is necessary to wrap download and execute together. There is no meaning in it. It should be left to the script function itself. There is no point to let all script functions call a command download/execute function. It is up to the script function itself to do their unique download and execution. This is better. Now you only see bash, there will be many other scripts in the future, download/execute function will decrease in its value along time elapse, although I do not think it has some value at the beginning. And I also mentioned many times, executing scripts are so diverse, there is no point to wrap execution and passing long trailing arguments again and improves nothing.
2. The correct thing to do is wrapping up individual scripts or multiple that serve the same purpose into their own function, then they perform their tasks inside. It is ridiculous to force their scripts to be in the same subfolder, let alone they are already in the folder and have their unique script names. Adding add layer of sub-folder improves nothing, it just requires you to do one more CD. This serves no purpose and nothing to do with any good or bad habits. It is just meaningless.